### DISCUSSION TITLE
Github Dev meme generator isn't working

### DISCUSSION THREAD

**Discussion by Mudassiruddin7** [VishwaGauravIn/github-profile-readme-maker#58] [General]
Github Dev meme generator isn't working from the past few weeks!!

**Comment by VishwaGauravIn** [comment]
Thanks for reporting, it has been fixed and you try it out now.

**Comment by Mudassiruddin7** [comment]
Again it's not working brother!!

**Comment by VishwaGauravIn** [comment]
I'll remove it from GPRM as it is not possible to run now for free on scale. 
Thanks for reporting 💛
